### DISCUSSION TITLE
Don't login each time container starts

### DISCUSSION THREAD

**Discussion by zoli** [bubuntux/nordvpn#321] [Q&A]
Is it possible to somehow keep nordvpn session (by sharing volumes maybe) so on each time container starts it doesn't login?

I'm working on a service which in current development state I need to stop, remove and again start the container multiple times. And after a while I face `logging in: default api: Too Many Requests` error. I want to prevent nordvpn login each time when the container starts just continue the last session.

**Comment by bubuntux** [comment]
try migrating to https://github.com/bubuntux/nordlynx
